src/pkey/clu_rsa.c

```diff
@@ -121,6 +127,15 @@ int wolfCLU_RSA(int argc, char** argv)
                 pubOnly = 1;
```

Comment:
Now that we have pubIn and pubOut, pubOnly is useless. You can set both pubIn and pubOut here for `WOLFCLU_RSAPUBIN ` and then remove pubOnly from the code (`WOLFCLU_RSAPUBIN` should do the same thing as `WOLFCLU_PUBIN` .